seml/main.py

```diff
@@ -182,11 +187,17 @@ def main():
     logging.root.setLevel(logging_level)
 
     if args.func == mongodb_credentials_prompt:  # launch SEML configure.
-        del args.config_file
+        del args.db_collection_name
     else:  # otherwise remove the flag as it is not used elsewhere.
-        if args.config_file is None:
+        if args.db_collection_name is None:
             logging.error("Please provide a path to the config file.")
             sys.exit(1)
+        else:
+            if os.path.isfile(args.db_collection_name):
```

Comment:
Is this the right way to do it? What if the user has a file that coincidentally has the same name?

Reply:
I've changed the warning to reflect this behavior. I think it's fine now.